src/journeys_utils.js

```diff
@@ -908,3 +908,72 @@ journeys_utils.setJourneyLinkData = function(linkData) {
 	}
 	journeys_utils.journeyLinkData = data;
 };
+
+journeys_utils.hasInBranchViewData = function(fieldName) {
```

Comment:
What does the name of the method mean?

Reply:
Ok, bad naming. :) The function returns **true** if in BranchViewData a field with the name: FieldName

Reply:
a better name could be `hasBranchViewData(fieldName) or isFieldInBranchViewData(fieldName) or  doesParameterExitsInBranchViewData(parameterName) or exitsBranchViewData(name)`

Reply:
`exitsValueInBranchViewData(name)`